wled00/FX.cpp

```diff
@@ -5665,11 +5664,11 @@ uint16_t mode_2DSindots(void) {                             // By: ldirko   http
     int y = sin8(t2 + i * SEGMENT.intensity/8)*(rows-1)/255;  // max index now 255x15/255=15!
     SEGMENT.setPixelColorXY(x, y, ColorFromPalette(SEGPALETTE, i * 255 / 13, 255, LINEARBLEND));
   }
-  SEGMENT.blur(SEGMENT.custom2>>1);
+  SEGMENT.blur(SEGMENT.custom2 >> (3 - (SEGMENT.check1<<1)));
 
   return FRAMETIME;
 } // mode_2DSindots()
-static const char _data_FX_MODE_2DSINDOTS[] PROGMEM = "Sindots@!,Dot distance,Fade rate,Blur;;!;2;c2=64";
+static const char _data_FX_MODE_2DSINDOTS[] PROGMEM = "Sindots@!,Dot distance,Fade rate,Blur,,Extra Blur;;!;2;";
```

Comment:
call it smear maybe? that would be the same as in other effects where you added it

Reply:
I am not happy with that name either... but smear is a different behaviour (it blurs without fading underlaying colors, causing a motion blurring or 'smear'). Well screw it, I will just update it to use smear (did not succeed in that before but I have an Idea)